server/testutil/testenv/testenv.go

```diff
@@ -206,3 +208,11 @@ func GetTestEnv(t testing.TB) *real_environment.RealEnv {
 
 	return te
 }
+
+type NoOpAtimeUpdater struct{}
+
+func (a *NoOpAtimeUpdater) Enqueue(_ context.Context, _ string, _ []*repb.Digest, _ repb.DigestFunction_Value) {
+}
+func (a *NoOpAtimeUpdater) EnqueueByResourceName(_ context.Context, _ string) {}
+func (a *NoOpAtimeUpdater) EnqueueByFindMissingRequest(_ context.Context, _ *repb.FindMissingBlobsRequest) {
+}
```

Comment:
This looks unused? (but if it is needed, it'd fit better in a separate `test_cache_proxy` package since it's not needed for most tests)